CHANGELOG.md

```diff
@@ -1,86 +1,116 @@
 # Changelog
 
+## [4.39.0](https://github.com/voxpupuli/beaker/tree/4.39.0) (2022-11-23)
+
+[Full Changelog](https://github.com/voxpupuli/beaker/compare/4.38.1...4.39.0)
+
+**Closed issues:**
+
+- Ruby 3.1/Psych 4 compatibility issues [\#1753](https://github.com/voxpupuli/beaker/issues/1753)
```

Comment:
Should we skip this since https://github.com/voxpupuli/beaker/pull/1758 was merged? And why isn't https://github.com/voxpupuli/beaker/pull/1758 in there?

Reply:
GCG still has some problems with beaker :( (https://github.com/github-changelog-generator/github-changelog-generator/issues/971). I  updated it by hand.